alot/db/manager.py

```diff
@@ -385,6 +385,9 @@ def get_threads(self, querystring, sort='newest_first'):
         assert sort in self._sort_orders
         q = self.query(querystring)
         q.set_sort(self._sort_orders[sort])
+        if exclude_tags:
```

Comment:
Hmm, for the default it shouldn't make a difference whether it's an empty list or an empty tuple though?

But mmm... I generally opt for explicit checks, but I can do it either way.  @pazz any opinions on this?